### PR TITLE
Add 'make update-webassets' script

### PR DESCRIPTION
**Pull request by awly**
Copying over from `teleport/ops` to make it easier to discover.
Also changed the script to clone the repos into a temp directory and
clean it up later. Without this, a nested `teleport` checkout would
cause all kinds of problems.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -641,3 +641,11 @@ update-vendor:
 	# create a symlink to the the original api package
 	rm -r vendor/github.com/gravitational/teleport/api
 	ln -s -r $(shell readlink -f api) vendor/github.com/gravitational/teleport
+
+# update-webassets updates updates the webassets repo using the latest webapps
```
Comment (webvictim) [inline Makefile:645]:
```suggestion
# update-webassets updates the minified code in the webassets repo using the latest webapps
```

**Review by webvictim — APPROVED** [review]

**Comment by russjones** [comment]
@alex-kovoy @kimlisa What do you two think?

**Comment by kimlisa** [comment]
I think its great, now i don't have to navigate to a different repo, i can stay in teleport which i have always opened

**Review by russjones — APPROVED** [review]
Bot.

**Pull request merged**